EIPS/eip-3754.md

```diff
@@ -0,0 +1,53 @@
+---
+eip: 3754
+title: A Vanilla Non-Fungible Token Standard
+description: A vanilla NFT standard by removing data fields and functions related to `URI`.
+author: Simon Tian <simon@dtopia.me>
```

Comment:
We prefer authors to use their github handle so our CI bot can tag them with notifications.
```suggestion
author: Simon Tian (@simontianx)
```